presidio-analyzer/README.MD

```diff
@@ -37,3 +37,11 @@ def analyze(self, text, entities, nlp_artifacts):
 The `analyze` method should return a list of [RecognizerResult](presidio_analyzer/recognizer_result.py). Refer to the [code documentation](presidio_analyzer/entity_recognizer.py) for more information.
 
 b. Reference and add the new class to the `RecognizerRegistry` module, in the `load_predefined_recognizers` method, which registers all code based recognizers.
+
+## Using Azure Text Analytics to detect PII data
+Presidio comes with a built-in `TextAnalyticsRecognizer`.
```

Comment:
Please let's add more details for people that are not familiar with the service. At the least explain that we're using the NER capability of the service, provide a link to https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/ and https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking?tabs=version-3#named-entity-recognition-versions-and-features and direct developers to the portal page where they can find the endpoint and key.

Reply:
I can make these changes if you like :)

Reply:
@CodeRunRepeat  please do

Reply:
Done, @itye-msft please check it out.